### PR TITLE
Replace "branch" labels in music files with local names

### PR DESCRIPTION
**Pull request by Rangi42**
Same as https://github.com/pret/pokecrystal/pull/670

### CODE REVIEW

File: audio/music/cinnabarmansion.asm [hunk 1]
```diff
@@ -94,11 +89,10 @@ Music_CinnabarMansion_branch_7ed6c::
 	note A#, 4
 	sound_ret
 
-
 Music_CinnabarMansion_Ch3::
 	note_type 12, 1, 1
-
-Music_CinnabarMansion_branch_7ed80::
+.mainloop:
+.introloop:
```
Comment (vulcandth) [inline audio/music/cinnabarmansion.asm:95]: Why the `.mainloop` and `.introloop` here? When in `Music_CinnabarMansion_Ch1::` you only used `.introloop`.

**Review by vulcandth — COMMENTED** [review]
Dannye's review will be far more helpful; as I am not as familiar with the Music code. However, I did go through looking for anything obvious that stood out to me. It all looks good. I only had one question. 👍

File: audio/music/cinnabarmansion.asm (hunk 1 above)
Comment (Rangi42) [reply audio/music/cinnabarmansion.asm:95]: Nice catch! `Music_CinnabarMansion_Ch1` should have a `.mainloop` too.

File: audio/music/finalbattle.asm [hunk 2]
```diff
@@ -331,8 +330,8 @@ Music_FinalBattle_branch_23429::
 	octave 4
 	note C#, 12
 	note_type 12, 4, 0
-	sound_call Music_FinalBattle_branch_23710
-	sound_call Music_FinalBattle_branch_23710
+	sound_call Music_FinalBattle_Ch2.sub2
+	sound_call Music_FinalBattle_Ch2.sub2
```
Comment (dannye) [inline audio/music/finalbattle.asm:334]: Not sure if I've ever noticed this reach-across before. Neat.

**Review by vulcandth — APPROVED** [review]

**Review by dannye — APPROVED** [review]

**Comment by dannye** [comment]
Do you want to knock out audio/sfx/battle_27.asm and audio/sfx/battle_36.asm too?

**Pull request merged**